kubernetes/container-azm-ms-agentconfig.yaml

```diff
@@ -176,6 +176,18 @@ data:
     [agent_settings.high_log_scale]
       enabled = false
 
+    # Retina Network Flow Logs throttle settings
+    # Controls the rate at which network flow log messages are processed. 
```

Comment:
comment to have inline with config parameters.
 # throttle_rate = 1200 # By default is 1000 and tune this value to control the amount of messages.

Reply:
addressed